proto/participation/events.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+package tendermint.spn.participation;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/tendermint/spn/x/participation/types";
+
+message EventAllocationsUsed {
+  string participant = 1;
+  uint64 auctionID = 2;
+  uint64 numAllocations = 3;
+}
+
+message EventAllocationsWithdrawn {
```

Comment:
We could maybe add `numAllocations` as well?

Reply:
The number of allocations withdrawn from the should always be the same `numAllocations` emitted in the corresponding `EventAllocationsUsed` message.  I think this would just be adding redundant data.